src/client/article.ts

```diff
@@ -215,12 +216,28 @@ function renderComments(): void {
 	}
 }
 
+function footerLinks(): void {
+	const privacySettingsLink = document.getElementById('ar-privacy-settings');
+	const privacyPolicyLink = document.getElementById('ar-privacy-policy');
```

Comment:
Would recommend using the `js-` prefix convention.

Reply:
ah yeah, true! thanks 👍